cedar-policy-validator/src/schema_file_format.rs

```diff
@@ -492,8 +511,10 @@ impl From<SchemaTypeVariant> for SchemaType {
     }
 }
 
-#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Serialize)]
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize)]
```

Comment:
wanna make sure this is ok with the language team, this is the only change that's not under the wasm feature flag

Reply:
No concerns, we haven't been strictly policing `derive`s.